packages/react/src/Restitution/HeaderRestitution.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { getComponentClassName } from "../Form/core";
+
+export type HeaderRestitutionProps = {
+  className?: string;
+  title: React.ReactNode;
+  subtitle?: React.ReactNode;
+  rightTitle?: React.ReactNode;
+  classModifier?: string;
+};
+
+export const HeaderRestitution = ({
+  title,
+  subtitle,
+  rightTitle,
+  className,
+  classModifier,
+}: HeaderRestitutionProps) => {
+  const componentClassName = getComponentClassName(
+    className,
+    classModifier,
+    "af-restitution__header",
+  );
+  return (
+    <header className={componentClassName}>
+      <div className="af-restitution__header-left">
+        <div className="af-restitution__title">
+          <span className="af-restitution__title-main">{title}</span>
+          {subtitle && (
+            <span className="af-restitution__title-subtitle">{subtitle}</span>
+          )}
```

Comment:
```suggestion
          {subtitle ? (
            <span className="af-restitution__title-subtitle">{subtitle}</span>
          ) : null}
```

Ca va éviter le bug nul où la personne passe une valeur nullable qui sera affichée (comme un `0` par exemple)